src/layouts/NoResult/styled.js

```diff
@@ -1,43 +1,29 @@
 import styled from "styled-components";
 
-export const Disclaimer = styled.span`
+export const Wrapper = styled.div`
+  text-align: center;
+  max-width: 1368px;
+  margin: 0 auto;
```

Comment:
This margin: 0 auto should be replace by margin, because when I search long phrase the Disclaimer is always trying to center all the inside text of his own and the searchText